capstone-app/src/app/modules/maps/map/map.component.ts

```diff
@@ -46,6 +46,965 @@ export class MapComponent implements OnInit, OnChanges {
     'Meal Takeaway', 'Mosque', 'Movie Theater', 'Museum','Night Club', 'Park', 'Police', 
     'Restaurant', 'Rv Park', 'Shopping Mall', 'Stadium', 'Store', 'Subway Station', 'Supermarket',
     'Synagogue', 'Taxi Stand', 'Tourist Attraction', 'Train Station', 'Transit Station', 'University', 'Zoo'];
+  darkMode: google.maps.MapTypeStyle[] = [
```

Comment:
I recommend exporting these from a separate style so as to not clutter the component file.